src/server/report/app.js

```diff
@@ -251,32 +207,22 @@ class App extends LitElement {
 	_handleFilterStatusChange(e) {
 		this._updateSearchParams({ status: e.target.value });
 	}
-	_handleOverlayChange(e) {
-		this._overlay = e.target.checked;
-	}
-	_handleTestClick(e) {
+	_handleListTestClick(e) {
 		this._updateSearchParams({ file: e.target.dataset.file, test: e.target.dataset.test });
 		return false;
 	}
-	_renderFile(file) {
-		const renderBrowserCell = (b) => {
-			if (!this._filterBrowsers.includes(b.name)) return nothing;
-			return html`<th>${b.name}</th>`;
-		};
+	_handleOverlayChange(e) {
+		this._overlay = e.target.checked;
+	}
+	_renderEmpty() {
```

Comment:
Hopefully Jeff can improve on this... 

<img width="346" alt="Screenshot 2023-07-18 at 5 22 15 PM" src="https://github.com/BrightspaceUI/testing/assets/5491151/102a20cf-a802-4f6a-aed4-869c2e43306e">

Reply:
Not possible